### PR TITLE
Add step definition "move rename" to smoke test with playwrigth

### PR DESCRIPTION
**Pull request by ScharfViktor**
This PR adds a new step definition: rename, accept, move, copy and add the ability to select a sharing role.
The test demonstrates the work of the new step definitions, in the future the test will be changed

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oCISLocks</strong> https://drone.owncloud.com/owncloud/web/19245/48/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/19245/50/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISResharing1</strong> https://drone.owncloud.com/owncloud/web/19245/60/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISTrashbinUploadMoveJourney</strong> https://drone.owncloud.com/owncloud/web/19245/63/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10NotificationBasic</strong> https://drone.owncloud.com/owncloud/web/19245/37/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternalUsers2</strong> https://drone.owncloud.com/owncloud/web/19245/56/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10Rename</strong> https://drone.owncloud.com/owncloud/web/19245/10/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingFilePermission</strong> https://drone.owncloud.com/owncloud/web/19245/18/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles1</strong> https://drone.owncloud.com/owncloud/web/19245/52/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10MoveUpload</strong> https://drone.owncloud.com/owncloud/web/19245/14/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISBasic</strong> https://drone.owncloud.com/owncloud/web/19245/47/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternalGroups</strong> https://drone.owncloud.com/owncloud/web/19245/54/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10iPhoneNotifications</strong> https://drone.owncloud.com/owncloud/web/19245/44/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsToRoot</strong> https://drone.owncloud.com/owncloud/web/19245/23/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10Resharing</strong> https://drone.owncloud.com/owncloud/web/19245/15/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles2</strong> https://drone.owncloud.com/owncloud/web/19245/53/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10Files2</strong> https://drone.owncloud.com/owncloud/web/19245/13/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10iPhone1</strong> https://drone.owncloud.com/owncloud/web/19245/45/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingNotifications</strong> https://drone.owncloud.com/owncloud/web/19245/38/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10Locks</strong> https://drone.owncloud.com/owncloud/web/19245/8/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingFolderAdvancedPermissionMU</strong> https://drone.owncloud.com/owncloud/web/19245/19/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersBlacklisted</strong> https://drone.owncloud.com/owncloud/web/19245/26/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternalUsers2</strong> https://drone.owncloud.com/owncloud/web/19287/56/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/19292/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Comment by ownclouders** [comment]
Results for <strong>oC10CreateDelete</strong> https://drone.owncloud.com/owncloud/web/19348/9/1
The following scenarios passed on retry:
- webUIDeleteFilesFolders/deleteFilesFolders.feature:275

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/19348/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

File: tests/smoke/support/api/http.ts [hunk 1]
```diff
@@ -8,18 +8,20 @@ export const request = async ({
...
   body,
-  user
+  user,
+  versionApi
 }: {
   method: 'POST' | 'DELETE' | 'PUT' | 'GET'
   path: string
   body?: BodyInit
   user?: User
+  versionApi: string
```
Comment (fschade) [inline tests/smoke/support/api/http.ts:18]: can we make v2.php as a default and versionApi optional (versionApi?)

File: tests/smoke/support/page/files/sharedWithMe.ts [hunk 2]
```diff
@@ -11,4 +12,24 @@ export class SharedWithMePage {
...
+    const startUrl = page.url()
+
+    this.navigate()
+    const objectExists = await cta.files.resourceExists({
+      page: page,
+      name: name
+    })
+
+    const statusResource = await page.textContent(
+      `//*[@data-test-resource-name="${name}"]/../../../../../td[3]/div[1]/span[1]`
```
Comment (fschade) [inline tests/smoke/support/page/files/sharedWithMe.ts:27]: the xpath "/../../../../../" looks a bit fragile to me, is there a better way of selecting the span?

**Review by fschade — COMMENTED** [review]
sone small findings, thanks for taking care

File: tests/smoke/support/api/http.ts (hunk 1 above)
Comment (ScharfViktor) [reply tests/smoke/support/api/http.ts]: @fschade thanks you for the review. I corrected it

File: tests/smoke/support/page/files/sharedWithMe.ts (hunk 2 above)
Comment (ScharfViktor) [reply tests/smoke/support/page/files/sharedWithMe.ts]: Yes, I agree with you. I fixed it ``//*[@data-test-resource-name="${name}"]/ancestor::tr//span[contains(@class, "file-row-share-status-text")]``

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingPermissions3</strong> https://drone.owncloud.com/owncloud/web/19416/59/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10iPhone1</strong> https://drone.owncloud.com/owncloud/web/19421/45/1
The following scenarios passed on retry:
- webUIDeleteFilesFolders/deleteFilesFolders.feature:59

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingPermissions2</strong> https://drone.owncloud.com/owncloud/web/19421/58/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by fschade — APPROVED** [review]
LGTM, if ci is happy 🚀

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsers</strong> https://drone.owncloud.com/owncloud/web/19432/25/1
The following scenarios passed on retry:
- webUISharingInternalUsersShareWithPage/shareWithUsers.feature:91

**Pull request merged**